src/Cake.Common.Tests/Fixtures/Tools/InnoSetupFixture.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Cake.Common.Tools.InnoSetup;
+using Cake.Core.IO;
+using Cake.Testing;
+using Cake.Testing.Fixtures;
+using NSubstitute;
+
+namespace Cake.Common.Tests.Fixtures.Tools
+{
+    internal sealed class InnoSetupFixture : ToolFixture<InnoSetupSettings>
+    {
+        private readonly IRegistry _registry;
```

Comment:
Even though it's not used, you can expose this property (with getter and setter). Makes it easy to access it if we need to add more tests.